cogstat/cogstat_stat.py

```diff
@@ -699,6 +699,168 @@ def variable_pair_standard_effect_size(data, meas_lev, sample=True, normality=No
                                                                                  classes="table_cs_pd"))
     return standardized_effect_size_result
 
+def calc_se_r2(r2, k, n):
+    """Calculate standard error of R-squared estimate.
+
+    Parameters
+    ----------
+    r2 : float
+        R-squared or adjusted R-squared point estimate.
+    k : int or float
+        Number of regressors in the model.
+    n : int or float
+        Number of observations.
+
+    Based on: Cohen, J., Cohen, P., West, S.G., and Aiken, L.S. (2003). Applied Multiple Regression/Correlation
+    Analysis for the Behavioral Sciences (3rd edition). Mahwah, NJ: Lawrence Earlbaum Associates. pp. 88
+    See also: https://stats.stackexchange.com/questions/175026/formula-for-95-confidence-interval-for-r2
+
+    Returns
+    -------
+    float
+    """
+
+    numerator = 4*r2*((1-r2)**2)*((n-k-1)**2)
```

Comment:
please, use spaces around operators (in the following lines too)

Reply:
Spaces added.